packages/core/src/components/FileUploader/File/File.tsx

```diff
@@ -138,15 +140,17 @@ export const HvFile = ({
 
   return (
     <li className={classes.root}>
-      {!hasError && inProgress && <span className={classes.progressbarBack} />}
-
       {!hasError && inProgress && (
-        <progress
-          className={classes.progressbar}
-          aria-valuemin={0}
-          aria-valuemax={100}
-          aria-valuenow={currentProgress}
-          style={{ width: `${currentProgress}%` }}
+        <HvProgressBar
+          classes={{
+            root: classes.progressbar,
+            progressBarContainer: cx(
```

Comment:
This was the less confusing way I found to keep the old classes and minimize breaking changes